4/index.js

```diff
@@ -1,18 +1,26 @@
 //lesson04--------------------------------------------------
+//chatGPT prompt："javascript DOM作成 li aタグにテキスト挿入「これです」 aタグの中にimgタグ 配列 複数 for文でループ"
+//reference:https://qiita.com/yrn03m/items/87e223acc6e733b50429    (参考）連想配列
 
 //create list item
 const createList = document.getElementById('js-list');
-const listItemLinkImgNames = ['bookmark','message'];
+const arrayItems = [
+  {to: "bookmark.html", img: "1.png", alt:"画像1", text: "ブックマーク"},
+  {to: "message.html", img: "2.png", alt:"画像2", text: "メッセージ"}];
+const fragmentItem = document.createDocumentFragment();
 
-for (let i = 0; i < listItemLinkImgNames.length; i++) { //for文でループさせる
+arrayItems.forEach(function (arrayItem) {
```

Comment:
soooo nits
アロー関数の方が簡単で読みやすいかもです! 
Approveなので修正不要です !
```suggestion
arrayItems.forEach(arrayItem =>  {
```

[参考](https://developer.mozilla.org/ja/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach#for_%E3%83%AB%E3%83%BC%E3%83%97%E3%81%8B%E3%82%89_foreach_%E3%81%B8%E3%81%AE%E5%A4%89%E6%8F%9B)

Reply:
無名関数を作るのでしたらそうですね

Reply:
お疲れ様です！
アロー関数ですね！
ありがとうございます。
まだこの辺あやふやなのでリファレンス見てまとめます✨
https://github.com/megusasuke0/frontendhandson/commit/ee51e4ca9a649d8739e37e38b458e0a5f00d4a94